app/assets/stylesheets/opportunities.scss

```diff
@@ -21,6 +21,10 @@ h1, h2 {
 .join-button {
   background-color: #FF851B;
   color: white;
+  &:hover {
+    background-color: #FF851B;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`